language/English/strings.po

```diff
@@ -8342,7 +8346,11 @@ msgctxt "#19178"
 msgid "Show channel information when switching channels"
 msgstr ""
 
-#empty string with id 19179
+#. Used as extension header on recordings window
+#: xbmc/pvr/windows/GUIWindowPVRRecordings.cpp
+msgctxt "#19179"
+msgid "Deleted recordings trash"
```

Comment:
yes, "Deleted" should be enough. We could go with something like "Recycler" to be in line with some operating systems, but "Deleted" is probably better in this particular case